.github/workflows/bpf-checks.yaml

```diff
@@ -13,6 +13,7 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
+        persist-credentials: false
```

Comment:
`persist-credentials` should be within the `with` block with the other arguments ;)